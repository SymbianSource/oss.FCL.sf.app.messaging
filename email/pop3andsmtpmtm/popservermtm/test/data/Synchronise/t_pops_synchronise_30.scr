+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password
+OK 12 18499
+OK 12 messages
1 1519
2 1598
3 1599
4 1559
5 1355
6 1519
7 1519
8 1598
9 1557
10 1558
11 1519
12 1599
.
+OK 12 messages
1 t_pops_synchronise:message026
2 t_pops_synchronise:message027
3 t_pops_synchronise:message028
4 t_pops_synchronise:message029
5 t_pops_synchronise:message030
6 t_pops_synchronise:message031
7 t_pops_synchronise:message032
8 t_pops_synchronise:message033
9 t_pops_synchronise:message034
10 t_pops_synchronise:message035
11 t_pops_synchronise:message036
12 t_pops_synchronise:message037
.
-ERR Unknown Command "CAPA"
+OK TOP 12 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message037; Tue, 14 Aug 2001 16:43:16 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aj2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e630080@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:49:26 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALV>; Tue, 14 Aug 2001 16:51:47 +0100
Message-ID: <t_pops_synchronise_message037@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 37
Date: Tue, 14 Aug 2001 16:43:59 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven
.
+OK TOP 11 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message036; Tue, 14 Aug 2001 15:57:09 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ac2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e626e90@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:49 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALT>; Tue, 14 Aug 2001 16:50:59 +0100
Message-ID: <t_pops_synchronise_message036@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 36
Date: Tue, 14 Aug 2001 16:43:11 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six Thirty-Six
.
+OK TOP 10 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message035; Tue, 14 Aug 2001 15:56:54 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ag2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e621e0f@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:28 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALR>; Tue, 14 Aug 2001 16:50:19 +0100
Message-ID: <t_pops_synchronise_message035@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 35
Date: Tue, 14 Aug 2001 16:42:31 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Five Thirty-Five Thirty-Five Thirty-Five Thirty-Five Thirty-Five
.
+OK TOP 9 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message034; Tue, 14 Aug 2001 17:10:21 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ah2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e61d473@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:48:09 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALP>; Tue, 14 Aug 2001 16:49:55 +0100
Message-ID: <t_pops_synchronise_message034@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 34
Date: Tue, 14 Aug 2001 16:42:07 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Four Thirty-Four Thirty-Four Thirty-Four Thirty-Four Thirty-Four
.
+OK TOP 8 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message033; Tue, 14 Aug 2001 16:18:19 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ak2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e6146ca@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:47:33 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAL3>; Tue, 14 Aug 2001 16:49:38 +0100
Message-ID: <t_pops_synchronise_message033@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 33
Date: Tue, 14 Aug 2001 16:41:50 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Three Thirty-Three Thirty-Three Thirty-Three Thirty-Three
.
+OK TOP 7 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message032; Tue, 14 Aug 2001 17:02:44 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ad2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e60ff12@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:47:15 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALN>; Tue, 14 Aug 2001 16:49:15 +0100
Message-ID: <t_pops_synchronise_message032@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 32
Date: Tue, 14 Aug 2001 16:41:27 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two Thirty-Two
.
+OK TOP 6 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message031; Tue, 14 Aug 2001 16:12:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ae2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e606802@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:46:36 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALL>; Tue, 14 Aug 2001 16:48:57 +0100
Message-ID: <t_pops_synchronise_message031@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 31
Date: Tue, 14 Aug 2001 16:41:09 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One Thirty-One
.
+OK TOP 5 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message030; Tue, 14 Aug 2001 16:46:34 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ab2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5f7dd1@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:45:36 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALG>; Tue, 14 Aug 2001 16:48:40 +0100
Message-ID: <t_pops_synchronise_message030@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 30
Date: Tue, 14 Aug 2001 16:40:52 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty Thirty 
.
+OK TOP 4 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message029; Tue, 14 Aug 2001 16:07:40 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id ai2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5dcbbc@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:45 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALA>; Tue, 14 Aug 2001 16:46:31 +0100
Message-ID: <t_pops_synchronise_message029@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 29
Date: Tue, 14 Aug 2001 16:38:42 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine Twenty-Nine
.
+OK TOP 3 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message028; Tue, 14 Aug 2001 17:13:53 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id al2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5d8482@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:43:27 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK0>; Tue, 14 Aug 2001 16:46:14 +0100
Message-ID: <t_pops_synchronise_message028@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 28
Date: Tue, 14 Aug 2001 16:38:26 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Eight Twenty-Eight Twenty-Eight Twenty-Eight Twenty-Eight
.
** Read cancelled
+OK disconnecting
