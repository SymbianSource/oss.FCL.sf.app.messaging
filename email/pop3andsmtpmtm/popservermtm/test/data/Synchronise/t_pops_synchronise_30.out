STAT
LIST
UIDL
CAPA
TOP 12 1
TOP 11 1
TOP 10 1
TOP 9 1
TOP 8 1
TOP 7 1
TOP 6 1
TOP 5 1
TOP 4 1
TOP 3 1
QUIT
